src/OSPSuite.Presentation/Presenters/SimulationOutputMappingPresenter.cs

```diff
@@ -127,24 +140,64 @@ private IEnumerable<DataRepository> allAvailableObservedDataUsedBySimulation()
       public void UpdateSimulationOutputMappings(SimulationOutputMappingDTO simulationOutputMappingDTO)
       {
          MarkSimulationAsChanged();
+
+         if (simulationOutputMappingDTO.Output.DisplayString.Equals(Captions.SimulationUI.NoneEditorNullText))
```

Comment:
yes. Use the variable instead of comparing display string. 
e.g. if(Equals(simulationOutputMappingDTO.Output, _noneEntry))

Reply:
true, but then I can find no way to unit test this. we can discuss if you want.

Reply:
hum... Not sure what you mean.  You need to test the behaviour, not the implementation .We can check it out tomorrow